sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -70,6 +85,15 @@ public void register(final @NotNull IHub hub, final @NotNull SentryOptions optio
     if (this.options.isEnableActivityLifecycleBreadcrumbs() || performanceEnabled) {
       application.registerActivityLifecycleCallbacks(this);
       this.options.getLogger().log(SentryLevel.DEBUG, "ActivityLifecycleIntegration installed.");
+
+      // this is called after the Activity is created, so we know if the App is a warm or cold
+      // start.
```

Comment:
`l`: Can you maybe elaborate a bit on why this is called after the Activity is created? Why don't we use `onActivityCreated` instead?

Reply:
I will add a better comment, but it's hard to explain and requires understanding deeply how the whole process start work, its a few pages that cant be easily resumed in one liner

Reply:
Can we maybe link to resources so you can find out on your own even when you have to read some articles?